EventHorizon/Indicator/AuraIndicator.lua

```diff
@@ -31,14 +31,16 @@ function AuraIndicator:new(target, start, stop, spellId, texture, config)
 	end
 	
 	self.ticks = {}
-
+	self.isExtended = false
+	self.adjustExtendedOffset = 0
+	self.originalDuration = stop - start
```

Comment:
this is already captured in the base class Indicator. You should be able to reference self.original.start and self.original.stop